tests/bin/initialize.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+npm run env run tests-wordpress "chmod -c ugo+w /var/www/html"
+npm run env run tests-cli "wp rewrite structure '/%postname%/' --hard"
+
+status=0
+npm run env run tests-cli "wp site list" || status=$?
+
+if [ $status -eq 0 ]
+then
+	echo "Multisite already initialized"
+else
+	echo "Converting to multisite"
```

Comment:
Do we need a multisite environment here? Not very much familiar with the plugin but based on functionality it seems we don't require a multisite test setup here. Please let me know if I am missing anything here. Thanks

Reply:
@iamdharmesh might be best to have you work on updates here and merge this in as you're able

Reply:
@jeffpaul raised new PR for this #83